### ISSUE TITLE
Oracle SQL*Plus SET FEEDBACK support is incomplete

### ISSUE THREAD

**Issue by alextercete** [flyway/flyway#2559]
###### Which version and edition of Flyway are you using?
Flyway 6.0.8

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)
CLI

###### Which database are you using (type & version)?
Oracle 18.3

###### Which operating system are you using?
Windows

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)

With `oracle.sqlplus=true`, ran the migration:

```sql
SET FEEDBACK ON
UPDATE practice SET modifieddate = sysdate;
```

###### What did you expect to see?

The output should contain `4 rows updated`.

###### What did you see instead?

No feedback.

**Comment by martinmlouis** [comment]
Hi,
Eagerly waiting for this feature. Also I hope we will get output like
Procedure Created
Package Body Created and so on for PL/SQL DB Objects

**Comment by MikielAgutu** [comment]
Note some usage of the `SET FEEDBACK` command is totally broken. A migration with:

`SET FEEDBACK 1;`

Gets an error:


`ERROR: Unable to parse statement in sql\V1.1__first.sql at line 1 col 1: Unclosed counted closure near index 7
[0-9]{1-5}`

**Comment by MikielAgutu** [comment]
Somewhat fixed by https://github.com/flyway/flyway/issues/2659 but we still need to improve support.

**Comment by hermanjustin** [comment]
+1 - I was trying to get the SET FEEDBACK ON to work properly and finally searched for bugs and found that one was open.  This would be much preferred to get the results of queries instead of running in debug -X mode with a filter.

**Comment by DoodleBobBuffPants** [comment]
@hermanjustin is `SET FEEDBACK ON` working properly for you now?

**Comment by hermanjustin** [comment]
Yes it was - released with 7.0 and works great!!!

**Comment by hermanjustin** [comment]
> @hermanjustin is `SET FEEDBACK ON` working properly for you now?

Yes it was - released with 7.0 and works great!!!

**Comment by DoodleBobBuffPants** [comment]
Glad to hear it!
